services/app-api/handlers/reports/create.ts

```diff
@@ -77,13 +77,16 @@ export const createReport = handler(async (event, _context) => {
 
   const julyMcparRelease = unvalidatedMetadata?.julyMcparRelease;
 
+  const topicXIIUpdate = unvalidatedMetadata?.topicXIIUpdate;
```

Comment:
...should change to `const topicXIIUpdate = unvalidatedMetadata?.topicXIIUpdate; || false;` and above line to`const julyMcparRelease = unvalidatedMetadata?.julyMcparRelease || false` to match typing.